dss/events/handlers/index.py

```diff
@@ -67,7 +67,7 @@ def create_index_data(s3, bucket, bundle_key, manifest, log):
     index_files = {}
     bucket = s3.Bucket(os.getenv("DSS_S3_TEST_BUCKET"))
     for file_info in files_info:
-        if file_info['indexed'] is True:
+        if file_info['indexed'].lower() == 'true':
```

Comment:
This does not correspond to the on-disk format of the bundles that I've seen. The on-disk format contains booleans, not strings. What's the rationale for changing it back here?